include/wil/winrt.h

```diff
@@ -1075,7 +1059,7 @@ class iterable_range
 };
 #pragma endregion
 
-#if defined(__WI_HAS_STD_VECTOR) || defined(WIL_DOXYGEN)
+#if (WIL_USE_STL && defined(WIL_ENABLE_EXCEPTIONS)) || defined(WIL_DOXYGEN)

```

Comment:
This condition is repeated a few times.  Would it be useful to define `WIL_USE_STL_WITH_EXCEPTIONS` or something like that?